src/helpers/applicationMessageError.ts

```diff
@@ -0,0 +1,19 @@
+import { ToastMeta } from '../app/notifications/types';
+
+export class ApplicationMesssageError extends Error {
```

Comment:
this could extend `CustomApplicationError`

Reply:
But don't we separate `CustomApplicationError`s from ones created with `makeApplicationError`?

Reply:
They are separate on purpose, however, maybe they should be named something like `ToastMessageError`?